### PR TITLE
testing

### PR DESCRIPTION
**Pull request by mattrobball**

### CODE REVIEW

**Comment by mattrobball** [comment]
!bench

File: Mathlib/Algebra/Category/GroupCat/Basic.lean [hunk 1]
```diff
@@ -134,7 +134,7 @@ set_option linter.uppercaseLean3 false in
 #align AddGroup.zero_apply AddGroupCat.zero_apply
 
 /-- Typecheck a `MonoidHom` as a morphism in `GroupCat`. -/
-@[to_additive]
+@[to_additive, nolint simpNF]
```
Comment (eric-wieser) [inline Mathlib/Algebra/Category/GroupCat/Basic.lean:137]: This one is weird too; `nolint simpNF` is surely a no-op on defs?

Comment (mattrobball) [reply Mathlib/Algebra/Category/GroupCat/Basic.lean:137]: Yes. I was aspirationally hoping it got applied to the generated lemmas.

**Comment by semorrison** [comment]
I have reset #7606 to match the state of this branch, and will no close this branch. Hopefully this is not too disruptive. This will make integrated CI with the Lean repository work better!

**Pull request closed without merging**